core/nodejs14Action/Dockerfile

```diff
@@ -25,15 +26,15 @@ RUN git clone --branch ${GO_PROXY_GITHUB_BRANCH} \
     mv proxy /bin/proxy
 
 # or build it from a release
-FROM golang:1.15 AS builder_release
+FROM $GO_PROXY_BASE_IMAGE AS builder_release
 ARG GO_PROXY_RELEASE_VERSION=1.15@1.17.0
 RUN curl -sL \
     https://github.com/apache/openwhisk-runtime-go/archive/{$GO_PROXY_RELEASE_VERSION}.tar.gz\
     | tar xzf -\
     && cd openwhisk-runtime-go-*/main\
     && GO111MODULE=on go build -o /bin/proxy
 
-FROM node:14.15.4-stretch
+FROM node:14-stretch
```

Comment:
this looks older?

Reply:
It's supposed to be the "latest release in the 14 line", which is 14.19.3